WordPress/gutenberg
# SelectControl: Add onChange, onBlur and onFocus to storybook actions

## What?
Log `onChange`, `onBlur` and `onFocus` props to the Actions screen in Storybook.

## Why?
To see what `onChange`, `onBlur` and `onFocus` does.

## How?
Update `SelectControl` storybook to log all props stating with `on` in the Actions screen.

## Testing Instructions
1. Fire up the storybook
2. Select the `SelectControl` story and change the value.
3. Confirm `onChange`, `onBlur` and `onFocus` was logged in the Actions screen.

## Screenshots or screencast 
![select-control-actions](https://user-images.githubusercontent.com/1415747/199273962-1b80bfe9-3be7-4a6d-9b3f-6fb98e026b6a.gif)
